### DISCUSSION TITLE
Can't install from AUR

### DISCUSSION THREAD

**Discussion by GBember01** [rocm-arch/rocm-arch#949] [General]
Hi! I know some packages were removed from the AUR because of the transition to the regular repo, but why remove them before they leave the testing repo to the stable repo? I think it might be dangerous to enable the testing repo, I would rather install those packages from the AUR.

**Comment by tpkessler** [comment]
Hey! ROCm will soon be moved from [community-testing] to [community].

> be dangerous to enable the testing repo

I would say it's not more dangerous than the stable repos.

**Reply by GBember01** [reply]
Do you have any estimative on when it will be released on the stable repo?

**Reply by tpkessler** [reply]
Packages have been released to [community] this morning!

**Reply by GBember01** [reply]
Wow! Thank you for your work!
